themes/_default.yaml

```diff
@@ -201,9 +201,10 @@ button:
       decoration: "underline"
 calendar:
   height: 300px
+  head:
+    border-radius: "5px 5px 0 0"
```

Comment:
before moving this variable here pls confirm that it is not overwritten in the travix-themes

Reply:
checked! It's not overwritten

Reply:
if it is not used at all pls remove the variable and add it directly to the sass file as value... :)